sentry-opentelemetry/sentry-opentelemetry-agentless-spring/README.md

```diff
@@ -0,0 +1,54 @@
+# sentry-opentelemetry-agentless-spring
+
+*NOTE: Our OpenTelemetry modules are still experimental. Any feedback is welcome.*
+
+## How to use it
+
+Add the latest `sentry-opentelemetry-agentless-spring` module as a dependency and add a `sentry.properties` 
+configuration file to your project that could look like this:
+
+```properties
+# NOTE: Replace the test DSN below with YOUR OWN DSN to see the events from this app in your Sentry project/dashboard
+dsn=https://502f25099c204a2fbf4cb16edc5975d1@o447951.ingest.sentry.io/5428563
+traces-sample-rate=1.0
+```
+
+For more details on configuring Sentry via `sentry.properties` please see the
+[docs page](https://docs.sentry.io/platforms/java/configuration/).
+
+As an alternative to the `SENTRY_PROPERTIES_FILE` environment variable you can provide individual
+settings as environment variables (e.g. `SENTRY_DSN=...`) or you may initialize `Sentry` inside
+your target application. If you do so, please make sure to apply OpenTelemetry specific options, e.g.
+like this:
+
+```
+Sentry.init(
+        options -> {
+          options.setDsn("...");
+          ...
+          OpenTelemetryUtil.applyOpenTelemetryOptions(options, false);
```

Comment:
This can be removed, as it is now obsolete. I suggest we do that in the #3978 PR